js/index.js

```diff
@@ -0,0 +1,198 @@
+let playTurn = 1;
+let countSpaces = 0;
+let AI;
+
+
+const gameBoard = (space, value) => {
```

Comment:
Note: The main goal of this project is for us to practice Factory Functions and The Module Pattern from the [previous section](https://www.theodinproject.com/courses/javascript/lessons/factory-functions-and-the-module-pattern) in the curriculum.
we should have `game`, `gameBoard`, and `player` objects
![image](https://user-images.githubusercontent.com/31897434/75406219-33b3ed80-5910-11ea-83e6-3d2be95b855d.png)